contentprovider.go

```diff
@@ -694,64 +709,64 @@ func scoreKind(language string, kind string) float64 {
 		// to "classes" instead of "c". We have to cover both cases to support existing
 		// indexes.
 		case "class", "classes":
-			factor = 10
+			score = Class
 		case "enum":
-			factor = 9
+			score = Enum
 		case "interface":
-			factor = 8
+			score = Interface
 		case "method":
-			factor = 7
+			score = Function
 		case "field":
-			factor = 6
+			score = Method
 		case "enumConstant":
-			factor = 5
+			score = Constant
 		}
 	case "Kotlin", "kotlin":
 		switch kind {
 		case "class":
-			factor = 10
+			score = Class
 		case "interface":
-			factor = 9
+			score = Enum
 		case "method":
-			factor = 8
+			score = Interface
```

Comment:
This confuses me more than before. I think it is ok that EG "interface" has a different score depending on the language. Interface might be a super important in one language, like Go, and a tiny bit less important in class based languages.

Reply:
I'd rather replace the strings with an enum and define mapping for both universal-ctags and scip-ctags to our internal representation.

Reply:
> This confuses me more than before. I think it is ok that EG "interface" has a different score depending on the language.

In case it wasn't clear, my proposal was to change this so that interfaces, etc. are treated the same across languages. Then this refactor will be clean and not so confusing :)

Anyways thanks for your thoughts, let's discuss more in our next meeting!

Reply:
Now that I'm up-to-speed on how this works, I understand and think we need a different approach. Closing this out.